guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -2,20 +2,20 @@
 = Preparing a Host for External Databases
 
 ifndef::orcharhino[]
-Install a freshly provisioned system with the latest {EL} 7 server to host the external databases.
+Install a freshly provisioned system with the latest {EL} 8 server to host the external databases.
 endif::[]
 ifdef::orcharhino[]
-Install a freshly provisioned system with the latest CentOS 7, Oracle Linux 7, or {RHEL} 7 to host the external databases.
+Install a freshly provisioned system with the latest CentOS 8 Stream, Oracle Linux 8, or {RHEL} 8 to host the external databases.
```

Comment:
@maximiliankolb can this also be simplified to `{EL} 8` and drop the whole ifdef or does Orcharhino want to explicitly mention some supported distros?

Reply:
EL8 is good for me for now! So yes, please drop the whole ifdef if that's the only diff.